crypto-core/build.gradle

```diff
@@ -11,8 +11,8 @@ dependencies {
     implementation 'com.palantir.safe-logging:preconditions'
     implementation 'com.palantir.safe-logging:safe-logging'
 
-    annotationProcessor 'org.openjdk.jmh:jmh-generator-annprocess'
-    compileOnly 'org.openjdk.jmh:jmh-generator-annprocess'
+    testAnnotationProcessor 'org.openjdk.jmh:jmh-generator-annprocess'
+    testCompileOnly 'org.openjdk.jmh:jmh-generator-annprocess'
```

Comment:
The benchmark lives in `src/test/java`

Reply:
maybe separate, but should we move them to `src/jmh/java` use use [jmh-gradle-plugin](https://github.com/melix/jmh-gradle-plugin)?

Reply:
Agree that would be cleaner, I've had some dependency issues with the gradle plugin that are probably solved or irrelevant at this point -- I usually create a separate module instead, but either way would be an improvement over reusing test sources.